### PR TITLE
Translated static keys to portuguese (Brazillian)

### PR DESCRIPTION
**Pull request by Packss**
I've added a portuguese translation for the static keys.

I never developed for android and never touched Kotlin but is there a way to translate the dynamic parts too?
This is a great project, I really missed a F1 app for Android TV.

### CODE REVIEW

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="login">Endereço de email</string>
```
Comment (leonardoxh) [inline app/src/main/res/values-pt/strings.xml:3]: This is too formal, maybe just `Email` ?

File: app/src/main/res/values-pt/strings.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <string name="password">Senha</string>
+    <string name="signin">Entrar</string>
+    <string name="signin_required_f1_tv_pro_account">Uma conta do F1Tv/F1TvPro é necessaria para login</string>
```
Comment (leonardoxh) [inline app/src/main/res/values-pt/strings.xml:6]: `necessária`

File: app/src/main/res/values-pt/strings.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <string name="invalid_credentials">Credenciais invalidas</string>
```
Comment (leonardoxh) [inline app/src/main/res/values-pt/strings.xml:7]: `inválidas`

**Review by leonardoxh — COMMENTED** [review]
Muito bom.

Hey I made a feel comments, I do not really know if they are right or wrong. But please go through them and I believe we can merge it.

Answering the other question, the dynamic content can be translated, if we do some tricks, the thing is that as we do not own the API if we start to do these things it might be that it changes on future and it generates too much work on our side to fix, and as this is a project that we only work on free time that is not that cool.

**Comment by bashopman** [comment]
@Packss regarding the dynamic parts, I had also translated those, but @leonardoxh mentioned correctly that those depend on the F1TV API. If they change it, it will break and will cause us work.
FYI: this commit contains a solution that works for me: https://github.com/bashopman/race-control-tv/commit/fa31e071e7a6e145fb220edf69ebe9ff225fe833

**Comment by Packss** [comment]
> Muito bom.
> 
> Hey I made a feel comments, I do not really know if they are right or wrong. But please go through them and I believe we can merge it.
> 
> Answering the other question, the dynamic content can be translated, if we do some tricks, the thing is that as we do not own the API if we start to do these things it might be that it changes on future and it generates too much work on our side to fix, and as this is a project that we only work on free time that is not that cool.

They are right actually, i didn't noticed it.

Totally get it really, the dynamic part at least in portuguese isn't really necessary, we say grand prix instead of grande prêmio anyway.

**Comment by Packss** [comment]
@bashopman yeah i agree, I just thought we could get the strings directly from the API, just like the titles of the videos (that are translated).

**Pull request merged**